test/transaction.test.js

```diff
@@ -71,6 +71,17 @@ describe('Transactions on test connector without execute()', () => {
     }, done);
   });
 
+  it('beginTransaction returns a transaction', done => {
+    const promise = db.beginTransaction(Transaction.READ_UNCOMMITTED);
+    promise.should.be.Promise();
+    promise.then(transaction => {
```

Comment:
Let's use await to keep the code simpler.

```suggestion
it('beginTransaction returns a transaction', async () => {
  // ...
  const transaction = await promise;
  // ...
}
```